components/category_menu_items/index.ts

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+export {default} from './category_menu_items';
```

Comment:
Do we need this connector?

Reply:
for import clarify and convention but I am ok with removing it.